hi-ml-azure/src/health_azure/utils.py

```diff
@@ -673,6 +673,26 @@ def download_checkpoint_if_necessary(self) -> Path:
         return self.local_checkpoint_path
 
 
+def get_checkpoint_downloader(ckpt_run_id: str, outputs_folder: Path) -> CheckpointDownloader:
+    """Creates an instance of CheckpointDownloader for the specified run_id and downloads the corresponding checkpoint.
+
+    :param ckpt_run_id: The Azure ML run_id of the run from which to load the checkpoint.
+    :param outputs_folder: The outputs folder where to download the checkpoint.
+    :return: A downloader instance for the specified run_id.
+    """
+    from health_ml.utils.common_utils import CHECKPOINT_FOLDER, DEFAULT_AML_UPLOAD_DIR
+    from health_ml.utils.checkpoint_utils import LAST_CHECKPOINT_FILE_NAME_WITH_SUFFIX
+    downloader = CheckpointDownloader(
+        aml_workspace=get_workspace(),
+        run_id=ckpt_run_id,
+        checkpoint_filename=LAST_CHECKPOINT_FILE_NAME_WITH_SUFFIX,
```

Comment:
That is a very good point, I will make that possible!

Reply:
@ant0nsc I'm not sure it makes sense to provide a checkpoint path under the `run_id` flag. Does it make more sense to have a separate argument for 'checkpoint_path'? (Which actually I think we should add in any case, since we currently assume the checkpoints are uploaded to a folder called 'outputs' which might not be the case for models that weren't trained with our pipeline)